README.md

```diff
@@ -9,7 +9,7 @@ This is not an official Google product.
 
 https://tensorflow.github.io/haskell/haddock/
 
-[TensorFlow.Core](https://tensorflow.github.io/haskell/haddock/tensorflow-0.1.0.2/TensorFlow-Core.html)
+[TensorFlow.Core](https://tensorflow.github.io/haskell/haddock/tensorflow-0.2.0.0/TensorFlow-Core.html)
```

Comment:
Do you think we can create a symlink and keep it up to date as a part of our [doc generation script](
https://github.com/tensorflow/haskell/blob/master/tools/haddock.sh)?

Reply:
👍